app/tests_unit.py

```diff
@@ -86,6 +86,20 @@ def test_phone_must_start_with_54(self):
         )
         self.assertFalse(saved)
         self.assertEqual(errors["phone"], "El teléfono debe comenzar con 54")
+        
+    def test_phone_must_be_only_numbers(self):
+        saved, errors = Client.save_client(
+            {
+                "name": "Juan Sebastian Veron",
+                "phone": "",
```

Comment:
this should be a string. Add validation in validate_client so the app doesn't break